File: bronze/2506/python/main.py

if (__name__ == "__main__"):
    num = int(input())

    arr = list(map(int, input().split()))
    score, count = (0, 0)
    for i in arr:
        count = (count +1) if (i) else 0
        score += count
    print(score)

File: bronze/2506/c++/main.cpp
#include <iostream>

int main()
{

    int num, score = 0, count = 0, temp;
    std::cin >> num;
    
    for (int i = 0; i < num; i++) {
        std::cin >> temp;
        if (temp) {
            count++;
            score += count;
        } else {
            count = 0;
        }
    }
    std::cout << score;

    return 0;
}